### PR TITLE
have the possibility to show a FeaturePopup at the geometry of the featu...

### PR DESCRIPTION
**Pull request by bartvde**
...re

### CODE REVIEW

**Comment by bartvde** [comment]
@marcjansen @chrismayer any objections to pulling this one in? I've verified that the tests and the existing featurepopup example still work as expected.

I'd like to use this in a new PR that will involve a simple feature editor.

File: examples/featurepopup-map.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            attribution : '&copy; terrestris GmbH &amp; Co. KG<br>Data &copy; OpenStreetMap contributors, CC-BY-SA'
+        });
+
+        map.addLayers([layerOwsOsmTerrestris, layerVector]);
+        
+        // A button to show relevant code parts
+        var btnSource = example.utility.getSourceCodeButton('featurepopup-map');
+        
+        // Create the viewport
+        var viewport = Ext.create('Ext.TabPanel', {
```
Comment (marcjansen) [inline examples/featurepopup-map.js]: I'd choose an `Ext.Panel` with `layout: 'fit'` instead.

File: examples/featurepopup-map.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            layers : 'OSM-WMS-Deutschland'
+        }, {
+            singleTile : false,
+            attribution : '&copy; terrestris GmbH &amp; Co. KG<br>Data &copy; OpenStreetMap contributors, CC-BY-SA'
+        });
+
+        map.addLayers([layerOwsOsmTerrestris, layerVector]);
+        
+        // A button to show relevant code parts
+        var btnSource = example.utility.getSourceCodeButton('featurepopup-map');
```
Comment (marcjansen) [inline examples/featurepopup-map.js:90]: That particular key `'featurepopup-map'` is not defined in the example-utility file. Could you please add a short description alongside some example code there?

File: examples/featurepopup-map.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+                new OpenLayers.Control.Navigation()
+            ]
+        });
+
+        var layerOwsOsmTerrestris = new OpenLayers.Layer.WMS('terrestris', 'http://ows.terrestris.de/osm-basemap/service', {
+            layers : 'OSM-WMS-Deutschland'
+        }, {
+            singleTile : false,
+            attribution : '&copy; terrestris GmbH &amp; Co. KG<br>Data &copy; OpenStreetMap contributors, CC-BY-SA'
+        });
```
Comment (marcjansen) [inline examples/featurepopup-map.js:85]: That WMS looks really neat in `EPSG:900913`. so if you'd find the time to switch the projection, that'd be extra cool.

**Comment by marcjansen** [comment]
I added some very minor comments, @bartvde. Please adress these and then feel free to merge. A great addition BTW.

File: examples/featurepopup-map.js (hunk 1 above)
Comment (bartvde) [inline examples/featurepopup-map.js]: Good point, too much copy/paste ;-)

File: examples/featurepopup-map.js (hunk 3 above)
Comment (bartvde) [inline examples/featurepopup-map.js:85]: I just copied the map from the existing Map example in GXM

**Pull request merged**